test/Conversion/ExportVerilog/hw-dialect.mlir

```diff
@@ -457,7 +458,7 @@ hw.module @signs(in %in1: i4, in %in2: i4, in %in3: i4, in %in4: i4)  {
   sv.assign %awire, %b4: i4
 
   // https://github.com/llvm/circt/issues/369
-  // CHECK: assign awire = 4'sh5 / -4'sh3;
+  // CHECK: assign awire = $signed(4'sh5 / -4'sh3);
```

Comment:
🥲 It's unfortunate that this shows up. However, to know that this can be elided would mean teaching `ExportVerilog` about context sensitive widths. This would be a huge pain and it's better to just do the safe, cheap thing. (If non-obvious, this is commentary and non-blocking.)